### ISSUE TITLE
The option 'pause when computer being used' is ignored

### ISSUE THREAD

**Issue by don-pit** [BOINC/boinc#1752]
Tasks are executed when user works on computer although corresponding option that pauses execution in such cases is selected.

Client version: 7.6.33 (x86)

**Comment by AenBleidd** [comment]
@don-pit Please check that option 'Activity'->'Run based on preferences' is also checked. Thanks

**Comment by Overtonesinger** [comment]
@don-pit , Please, would You be so kind to provide steps to reproduce?Thank You so much! Because I want to investigate this.
And I have a few questions about actions with **high-risk** of destroying the local settings of computation - or of avoiding their proper check:

1. Which projects is the PC connected to - and which project was added as the last one?
2. Did You *ever* (in the whole existence of the installation of BOINC) press the button "use web preferences" (from any project) - and especially lately - before this issue happened?
3. Was the BOINC "snoozed" (for one hour) and then immediatelly the PC hibernated for EXACTLY "59 minutes +59 seconds" before it woke up? If YES, there's a HIGH risk of the "wakeup+autoUnsnooze after 1 hour" issue that I once or twice encountered myself. (I used to travel to work exactly 1 hour,  door to door).

**Comment by don-pit** [comment]
@Overtonesinger , An issue is detected right after upgrading from v7.6.22. When setup had done, I launched manager and (after UAC check) found running tasks. After reboot the behavior is normalized and the option seems to work properly.

Question (to comply with formalities):
1. The PC is connected to SETI@home project only.
2. No, I did not select "use web preferences".
3. No, I rarely use hibernation.

Since behavior is normalized I could close an issue.
